app/templates/views/broadcast/preview-areas.html

```diff
@@ -28,6 +28,7 @@
   {% for area in broadcast_message.areas %}
     {% if loop.first %}
       <ul class="area-list">
+        <p class="govuk-!-margin-right-2">Area: </p>
```

Comment:
Same here `<p>` isn't a valid child element of `<ul>`, perhaps make a new `<li class="area">` or move the `<p>` outside the `<ul>` ?

Reply:
Don't know why I missed that. Amended.